ci/check-closing-quotes.yml

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
What is the purpose of this file? It seems already inline in `azure-pipelines.yml`, and I can't think of a reason to try and factor it out.

Reply:
good catch! removing.
(first I thought we need this, forgot to remove)